sklearn/tests/test_calibration.py

```diff
@@ -395,11 +395,13 @@ def test_sigmoid_calibration():
         _SigmoidCalibration().fit(np.vstack((exF, exF)), exY)
 
 
+# TODO(1.3): Remove warning filter and normalize parameter.
+@pytest.mark.filterwarnings("ignore::FutureWarning")
 def test_calibration_curve():
     """Check calibration_curve function"""
     y_true = np.array([0, 0, 0, 1, 1, 1])
     y_pred = np.array([0.0, 0.1, 0.2, 0.8, 0.9, 1.0])
-    prob_true, prob_pred = calibration_curve(y_true, y_pred, n_bins=2)
+    prob_true, prob_pred = calibration_curve(y_true, y_pred, n_bins=2, normalize=False)
```

Comment:
we should not need to set `normalize=False` at this point and just be sure that we adapt the test such that we don't raise any `FutureWarning`.

Reply:
I agree with you here. I only included this for branch coverage to satisfy the Codecov check which otherwise fails.